lib/genesis_collector/collector.rb

```diff
@@ -106,7 +106,7 @@ def collect_memories
         else
           empty = m['size'] == 'No Module Installed'
           {
-            size: m['size'].to_i * 1000000,
+            size: m['size']['GB'] ? m['size'].to_i * 1000000000 : m['size'].to_i * 1000000,
```

Comment:
It's kind of confusing since it's a mixture of mebibytes and megabytes, as well as gibibytes and gigabytes.

In this case though we've done the one level multiplication of mebibytes, so we should follow that for gibibytes for proper reporting.